pkg/kopia/command/storage/secret_utils.go

```diff
@@ -0,0 +1,48 @@
+package storage
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+type locType string
+
+const (
+	locTypeS3        locType = "s3"
+	locTypeGCS       locType = "gcs"
+	locTypeAzure     locType = "azure"
+	locTypeFilestore locType = "filestore"
+)
+
+const (
+	bucketKey        = "bucket"
+	endpointKey      = "endpoint"
+	prefixKey        = "prefix"
+	regionKey        = "region"
+	skipSSLVerifyKey = "skipSSLVerify"
+	typeKey          = "type"
+)
+
```

Comment:
As suggested above, we would also need a utils file to read this info from files 🤔